tasks/create-github-release/README.md

```diff
@@ -13,6 +13,10 @@ a `release` dir.
 | repository | The github repository to release to | No | - |
 | release_version | The version string to use creating the release | No | - |
 | content_directory | The directory inside the workspace to find files for release | No | - |
+| resultsDirPath | Path to results directory in the data workspace | No | - |
+
+## Changes in 1.1.0
```

Comment:
The new parameter is mandatory, so it's a breaking change and should be 2.0.0.

Reply:
Updated